samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -725,7 +726,7 @@ class SamzaContainer(
       shutdownDiskSpaceMonitor
       shutdownHostStatisticsMonitor
       shutdownProducers
-      shutdownLocalityManager
+      shutdownCoordinatorStreamManager
```

Comment:
The fact that the start/shutdown sequence is asymmetric let me think that it would be better to fold this two steps.